components/alert/alert.js

```diff
@@ -69,33 +68,28 @@ class Alert extends LocalizeCoreElement(RtlMixin(LitElement)) {
 			}
 
 			.d2l-alert-highlight {
-				border-radius: 0.3rem 0 0 0.3rem;
-				bottom: 0;
-				left: 0;
-				margin: -1px;
-				min-width: 0.3rem;
+				border-end-start-radius: 0.3rem;
+				border-inline-start-style: solid;
+				border-inline-start-width: 0.3rem;
+				border-start-start-radius: 0.3rem;
+				inset-block-end: -1px;
+				inset-block-start: -1px;
+				inset-inline-start: -1px;
```

Comment:
This made me wonder why we didn't just apply a 0.3rem left border to the main container in the first place. [Trying that out here](https://github.com/BrightspaceUI/core/pull/5488), which seems promising and simpler.

Reply:
Yeah that is weird... so much simpler.